app/assets/components/cookies/cookies.js

```diff
@@ -0,0 +1,116 @@
+export default () => {
+    class Cookies {
+        constructor() {
+            // banner
+            this.banner = document.querySelector('.nhsuk-cookie-banner')
+            this.showCookies = document.getElementById('showCookies')
+            this.removeCookies = document.getElementById('removeCookies')
+            this.host = this.getHost()
+
+            this.setCookies()
+            this.addEventListeners()
+
+            // policy page
+            this.Status = document.querySelector('.nhsuk__cookieStatus')
+            this.In = document.querySelectorAll('.nhsuk__cookiesIn')
+            this.Out = document.querySelectorAll('.nhsuk__cookiesOut')
+
+            this.toggleStatus()
+        }
+
+        getHost(){
+          const host = window.location.host.toString().split(":")[0]
+          return host
+        }
+
+        addEventListeners() {
+            if(this.showCookies) {
+                this.showCookies.addEventListener('mousedown', this.showCookie)
+            }
+            if(this.removeCookies) {
+                this.removeCookies.addEventListener('mousedown', this.removeCookie)
+            }
+        }
+
+        bannerShow() {
+            this.banner.style.display = "block"
+        }
+        
+        bannerHide() {
+            this.banner.style.display = "none"
+        }
```

Comment:
Hi David - Can you check for nulls on showCookies and removeCookies please, to handle the scenario where there are is no cookie banner on the page.